ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/sync/fastsync/FastSyncTargetManager.java

```diff
@@ -91,6 +91,7 @@ public FastSyncTargetManager(
         .thenApply(
             result -> {
               if (peerHasDifferentPivotBlock(result)) {
+                LOG.warn("Best peer has wrong pivot block, disconnect: {}", bestPeer);
```

Comment:
This is a big deal right?  So warn is appropriate?

Reply:
Yeah, could indicate we synced to the wrong fork.